proxy/http/HttpTransact.cc

```diff
@@ -3571,11 +3571,7 @@ HttpTransact::handle_response_from_parent(State *s)
       ink_assert(s->hdr_info.server_request.valid());
 
       s->current.server->connect_result = ENOTCONN;
-      // only mark the parent down in hostdb if the configuration allows it,
-      // see proxy.config.http.parent_proxy.mark_down_hostdb in records.config.
-      if (s->txn_conf->parent_failures_update_hostdb) {
-        s->state_machine->do_hostdb_update_if_necessary();
-      }
+      s->state_machine->do_hostdb_update_if_necessary();
```

Comment:
yeah it is, see https://github.com/apache/trafficserver/pull/1464 merged about a week ago.